core/trino-main/src/test/java/io/trino/operator/BenchmarkGroupByHash.java

```diff
@@ -93,12 +100,34 @@ public Object groupByHashPreCompute(BenchmarkData data)
         return pageBuilder.build();
     }
 
+    @Benchmark
+    @OperationsPerInvocation(POSITIONS)
+    public List<Page> benchmarkHashPosition(BenchmarkData data)
```

Comment:
nit: it seems like a random place to benchmark `InterpretedHashGenerator` since it's also used in other places (e.g `PartitionLookupSource`). IMO a separate class would be better (easier to find when changing code). Otherwise people might forget that `InterpretedHashGenerator` is benchmarked here.

Reply:
I agree, but this was the benchmark that existed and making a new class would require a duplicating the setup / data generation code quite a bit. Is moving this benchmark to another class a blocker to PR merging or can we make that a `TODO` item for the future?

Reply:
nits are not blockers generally